webcompat/static/js/lib/bugform.js

```diff
@@ -17,7 +17,6 @@ function BugForm() {
   this.submitButtons = $("#js-ReportForm .js-Button");
   this.submitTypeInput = $("#submit_type:hidden");
   this.uploadLabel = $(".js-label-upload");
-  this.uploadLoader = $(".js-image-loader");
```

Comment:
@Regaddi what's the thinking for removing this? Wont' we have no feedback during an upload?

Reply:
SGTM. Thanks.